### ISSUE TITLE
FEATURE: Remove reward for new members - LOW PRIO

### ISSUE THREAD

**Issue by martinskalicky** [recursivetree/seat-transport#4]
Hello,

we do support new members with free cargo for first 2 weeks.

If plugin will be able to read how long is the guy in corporation (settings) and if its less then X (settings) set reward to 0 (settings)

Martin

**Comment by recursivetree** [comment]
I will consider it, but it is rather low on my priority list, as it is a highly specific feature. You can also put it in the info text and if you see a 0 isk contract, open the info window and check before accepting

**Comment by martinskalicky** [comment]
Yes, this is low priority. It's like nice to have rather then critical feature.
